bigquery/samples/tests/test_get_dataset_labels.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .. import get_dataset_labels
+
+
+def test_get_dataset_labels(capsys, client, dataset_id, dataset_with_labels_id):
+
+    get_dataset_labels.get_dataset_labels(client, dataset_id)
```

Comment:
Shouldn't this be `dataset_with_labels_id`? The `dataset_id` fixture is not needed, right?